tests/foreman/cli/test_oscap.py

```diff
@@ -57,16 +57,26 @@ def test_positive_list_default_content_with_admin(self, module_target_sat):
             1. Login to shell from admin account.
             2. Execute the scap-content command with list as sub-command.
 
-        :expectedresults: Default scap-content are listed.
+        :expectedresults: Default scap-content are listed. They should already be populated on a freshly installed Satellite.
 
         :BZ: 1749692
+        :verifies: SAT-30938
 
         :customerscenario: true
 
         :CaseImportance: Medium
         """
-        scap_contents = [content['title'] for content in module_target_sat.cli.Scapcontent.list()]
-        assert f'Red Hat rhel{module_target_sat.os_version.major} default content' in scap_contents
+        scap_contents = [content['title'] for content in satellite_host.cli.Scapcontent.list()]
+        for rhel_version in range(6, 9):
```

Comment:
I'm a bit rusty with python, but wouldn't this check only 6, 7 and 8?

Reply:
Yes!
```
>>> list(range(6,9))
[6, 7, 8]
```
Thanks

Reply:
Is this intended to not choose EL10 here? Do we have no scap contents for EL10 yet?

Reply:
@Gauravtalreja1 see the top-level comment